src/System Application/App/Feature Key/src/FeatureManagementImpl.Codeunit.al

```diff
@@ -560,4 +560,19 @@ codeunit 2610 "Feature Management Impl."
             RenameFeatureDataUpdateStatus.Rename(RenameFeatureDataUpdateStatus."Feature Key", Rec.Name);
         until FeatureDataUpdateStatus.Next() = 0;
     end;
-}
+
+    [EventSubscriber(ObjectType::Codeunit, Codeunit::"Feature Management Triggers", OpenFeatureManagement, '', false, false)]
+    local procedure DefaultOpenFeatureManagement()
+    var
+        FeatureManagementFacade: Codeunit "Feature Management Facade";
+        IsHandled: Boolean;
+        FeatureManagementPageID: Integer;
+    begin
+        FeatureManagementPageID := Page::"Feature Management";
```

Comment:
Not sure we actually need this line

Reply:
Seems redundant since the subscriber would be overwriting the value. No harm either though.